app/models/course/material.rb

```diff
@@ -0,0 +1,5 @@
+class Course::Material < ActiveRecord::Base
```

Comment:
This was the name I was using before exactly, but got a name conflict with global `File`.